app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallScreen.kt

```diff
@@ -131,7 +135,16 @@ fun OngoingCallScreen(
         flipCamera = sharedCallingViewModel::flipCamera,
         setVideoPreview = sharedCallingViewModel::setVideoPreview,
         clearVideoPreview = sharedCallingViewModel::clearVideoPreview,
-        onCollapse = { activity.moveTaskToBack(true) },
+        onCollapse = {
+            if (BuildConfig.PICTURE_IN_PICTURE_ENABLED && isPiPAvailableOnThisDevice) {
```

Comment:
As this verification logic is used in more places, can't we have it as a variable at the top as well?

Something like:
```
val shouldUsePiPMode = BuildConfig.PICTURE_IN_PICTURE_ENABLED && isPiPAvailableOnThisDevice

then use it as:
if (shouldUsePiPMode) { .. }
```